modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/struts/WebSsoAction.java

```diff
@@ -63,6 +61,9 @@ protected String doExecute(
 		return null;
 	}
 
+	@Reference(unbind = "-")
```

Comment:
@W2jx Usually we don't add `unbind = "-"` to field reference, please clean it up.

Reply:
Thank you, I will fix it.